tests/test_minpvprocessor.cpp

```diff
@@ -59,21 +59,21 @@ BOOST_AUTO_TEST_CASE(Pinch)
          3.5, 3.5, 3.5, 3.5,
          6, 6, 6, 6
         };
-    auto minpv_result = mp1.process(thickness, z_threshold, pv, minpvv, actnum, fill_removed_cells, z1.data(), pinch_no_gap);
+    auto minpv_result = mp1.process(thickness, z_threshold, 1e20, pv, minpvv, actnum, fill_removed_cells, z1.data(), pinch_no_gap);
```

Comment:
I suggest you add a test where maxgap actually make a difference.

Reply:
Good point. I added one